dnsclient.go

```diff
@@ -11,16 +11,27 @@ import (
 
 const hexDigit = "0123456789abcdef"
 
+var DefaultClient = &Client{resolver: net.DefaultResolver}
```

Comment:
```suggestion
var DefaultDNSClient = &DNSClient{resolver: net.DefaultResolver}
```

Let's call it DNSClient for clarity with BulkClient.